explorer/trace_testdata/full_trace.carbon

```diff
@@ -13,6 +13,42 @@
 // CHECK:STDOUT:
 // CHECK:STDOUT: }
 // CHECK:STDOUT: ********** resolving names **********
+
+
+package ExplorerTest api;
+
+// CHECK:STDOUT: --- add exposed name `interface TestInterface` (full_trace.carbon:[[@LINE+2]])
+// CHECK:STDOUT: --- declared `TestInterface` in `interface TestInterface` (full_trace.carbon:[[@LINE+1]])
+interface TestInterface {}
+
+fn Main() -> i32 {
+  return 0;
```

Comment:
This test doesn't actually resolve any names, so you're not getting good coverage of your name resolution additions here. Maybe we could add a little bit more code to this file so that we can see name resolution happening:

```suggestion
package ExplorerTest api;

// CHECK:STDOUT: --- add exposed name `interface TestInterface` (full_trace.carbon:[[@LINE+2]])
// CHECK:STDOUT: --- declared `TestInterface` in `interface TestInterface` (full_trace.carbon:[[@LINE+1]])
interface TestInterface {}

fn F(n: i32) -> i32 {
  return n + 1;
}

fn Main() -> i32 {
  return F(0);
```

... should do some semi-interesting name resolution.